package.json

```diff
@@ -12,15 +12,32 @@
     "url": "https://github.com/eventualbuddha/keysim.js"
   },
   "scripts": {
-    "build": "gobble build dist -f",
-    "test": "npm run test:node && npm run test:browser",
-    "test:browser": "./node_modules/karma/bin/karma start karma.conf.js",
-    "test:node": "mocha -R spec --recursive test"
+    "build": "browserify . --standalone keysim > dist/keysim.js",
```

Comment:
I'd prefer to build with `rollup` instead as it creates a smaller build and is faster. I realize it's new compared to browserify and doesn't work as out-of-the-box due to working with ES2015 syntax, but I think it has a brighter future.